config/install-latest-testing.yaml

```diff
@@ -207,7 +207,6 @@ metadata:
     app.kubernetes.io/instance: kyverno
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: kyverno-crds
-    app.kubernetes.io/version: v0.0.0
```

Comment:
Why the label is removed here?

Reply:
Bump @KhaledEmaraDev ⬆️

Reply:
@realshuting It's auto generated from codegen. Investigating why.

Reply:
@realshuting Helm doesn't accept `latest`. It wants a semantic version. This is because the other end of the if statement is `appVersion`.

If we make both strings it could work.